src/main/java/com/aws/greengrass/clientdevices/auth/api/UseCases.java

```diff
@@ -6,26 +6,26 @@
 package com.aws.greengrass.clientdevices.auth.api;
 
 import com.aws.greengrass.dependency.Context;
+import com.aws.greengrass.logging.api.Logger;
+import com.aws.greengrass.logging.impl.LogManager;
 import com.aws.greengrass.util.CrashableFunction;
 
 
 public class UseCases {
-    private final Context context;
-    private static UseCases instance;
+    private Context context;
+    public static final Logger logger = LogManager.getLogger(UseCases.class);
```

Comment:
public seems wrong

Reply:
I didn't wan't to create a logger per usecase, but it doesn't seem that bad to do it.